src/controls/tab-bar/focus.js

```diff
@@ -1,33 +1,40 @@
-import { KeyCodes } from '../../utils'
+const Keys = {
+  HOME: 'Home',
+  END: 'End',
+  LEFT: 'ArrowLeft',
+  RIGHT: 'ArrowRight',
+  UP: 'ArrowUp',
+  DOWN: 'ArrowDown',
+}
 
-// Funnction that can be passed to event handlers (e.g., onKeyPress) that manages which element should be focused
+// Function that can be passed to event handlers (e.g., onKeyPress) that manages which element should be focused
 // Note: Expected keyboard interaction with arrow keys changes depending on the orientation of the tab list
-function manageFocus(e, { vertical = false }) {
+function manageFocus(e, { vertical }) {
   // If not activated while on a tab, then ignore
   if (!isTabControl(e.target)) return
 
-  const key = (e.which || e.keyCode || '').toString()
+  const key = e.key === 'Unidentified' ? e.code : e.key
```

Comment:
If `key` cannot be identified, it returns `Unidentified`

Source: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key#value

Reply:
TIL 😃
Would it help to have a comment in the code?